examples/async-std.rs

```diff
@@ -1,14 +1,14 @@
+use async_std::net::TcpStream;
 use once_cell::sync::Lazy;
 use std::env;
 use tiberius::{Client, Config};
-use async_std::net::TcpStream;
 
 static CONN_STR: Lazy<String> = Lazy::new(|| {
-    env::var("TIBERIUS_TEST_CONNECTION_STRING")
-        .unwrap_or_else(|_| "server=tcp:localhost,1433;IntegratedSecurity=true;TrustServerCertificate=true".to_owned())
```

Comment:
why not unwrap_or()?

Reply:
This version keeps clippy happier :)

Reply:
`to_owned` is allocating memory and if using `unwrap_or`, it'll allocate even if we don't need it.

`unwrap_or_else` gets a lambda that allocates with the default value only if the env var is not set.

Reply:
> `to_owned` is allocating memory and if using `unwrap_or`, it'll allocate even if we don't need it.
> 
> `unwrap_or_else` gets a lambda that allocates with the default value only if the env var is not set.

I had no idea of this distinction! Thank you!